perfkitbenchmarker/linux_benchmarks/fio_benchmark.py

```diff
@@ -394,7 +396,9 @@ def GetLogFlags():
                    (FLAGS.fio_iops_log, '--write_iops_log=%(filename)s',),
                    (collect_logs, '--log_avg_msec=%(interval)d',)]
   fio_command_flags = ' '.join([flag for given, flag in fio_log_flags if given])
-  return fio_command_flags % {'filename': PKB_FIO_LOG_FILE_NAME,
+  now = time.time()
+  filename_base = '%s_%s' % (PKB_FIO_LOG_FILE_NAME, str(now))
```

Comment:
Good point about converting to nanoseconds. However, since you don't feel strongly, I would rather leave it as-is because I think I am more likely to recognize a timestamp in seconds, with decimal, if I am looking at filenames six months from now and can't remember the context.